reporter/src/main/kotlin/gitlab/GitLabLicenseModelMapper.kt

```diff
@@ -53,7 +53,7 @@ private fun Collection<Package>.toLicenses(): List<License> {
         License(
             id = singleLicenseExpression.simpleLicense(),
             name = singleLicenseExpression.toLicenseName(),
-            url = "" // TODO
+            url = if (singleLicenseExpression.isValid()) singleLicenseExpression.getLicenseUrl() else ""
```

Comment:
commit: I wonder if we should really integrate this `URL` thing into the `Expression`. An easy way to decouple things a bit would be to write e.g. 
```
LicenseUrlResolve : object {
   fun getUrl(license: SpdxSingleLicenseExpression): URL? {
       // resolve URLs for SPDX licenses as well as for all `LicenseRef-*` licenses ORT know about.
   }
```
This would not add another reponsibility to `SpdxExpression` which already has enough I believe. What do you think?

Reply:
So far I kept it in `SpdxExpression` with changes proposed above. Not sure if it's better to move it completely to `GitLabLicenseModelMapper.kt`...
@sschuberth  @mnonnenmacher what do you think?